internal/ignition/boot-reporter/assisted-boot-reporter.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
```

Comment:
Let's run lint this file with shellcheck to avoid common mistakes

Reply:
done

Reply:
done

Reply:
/hold

Reply:
I don't see a shellcheck linter being called anywhere so holding